utils/genesis/src/lib.rs

```diff
@@ -103,14 +103,9 @@ where
             )
         })?;
 
-    let store_genesis = chain_store.genesis()?;
+    let store_genesis = chain_store.genesis();
 
-    if store_genesis
-        .map(|store| store == genesis_block)
-        .unwrap_or_default()
-    {
-        debug!("Genesis from config matches Genesis from store");
-    } else {
+    if store_genesis.is_err() {
```

Comment:
You can delete `store_genesis` altogether. We always want to update the genesis in the DB.

Reply:
Deleted